packages/jaeger-ui-components/src/TraceTimelineViewer/TimelineHeaderRow/TimelineCollapser.tsx

```diff
@@ -56,40 +36,27 @@ type CollapserProps = {
   onExpandAll: () => void;
 };
 
-function getTitle(value: string) {
+export default function TimelineCollapser(props: CollapserProps) {
```

Comment:
Out of curiosity why isn't this declared as a functional component? Also I think we try to avoid default exports.

Reply:
What do you mean? It is a functional component.

Reply:
Hmm, I thought we had a style guideline for React functional components to be like `TimelineCollapser: FC<CollapserProps>` but we don't so my bad.

Reply:
To clarify I knew this is a functional component but I didn't see the specific FC return type so that's why I asked because I thought we did have a style guideline :)

Reply:
@hugohaggmark I remember that when I was using `const Component: FC<CollapserProps> = () => {...}` It did not pick up the component name from the variable name (like in the react dev tools for example) and I had to manually add `displayName` which was a bit tedious. Is that still happening or was that fixed in some way?

Reply:
@aocenas thanks, I've seen so many const Compontent: FC<...> lately I just thought that was our standard.